nucypher/cli/painting.py

```diff
@@ -420,10 +420,11 @@ def paint_stakes(emitter, stakes, paint_inactive: bool = False):
         emitter.echo(header, bold=True)
         emitter.echo(breaky, bold=True)
 
-        for stake in sorted([s for s in stakes if s not in active_stakes],
+        for stake in sorted([s for s in stakes if s not in active_stakes],  # TODO
```

Comment:
TODO comment - what do you envision needing to do?

Reply:
Ill make an update for it - I have a vision for some code optimization here, albeit perhaps premature.

Reply:
:heavy_check_mark: